src/components/UiLibrary.js

```diff
@@ -47,6 +48,22 @@ const UiLibrary = () => {
   `.trim();
   return (
     <ViewWithFooter>
+      <FloatingActionBar position="bottomStart" marginX="mx-4" marginY="my-11">
+        <View className="m-1 flex flex-row">
+          <IconButton
+            icon="pencil"
+            mode="contained"
+          />
+          <IconButton
+            icon="plus-sign"
+            mode="contained"
+          />
+          <IconButton
+            icon="icon-fave"
+            mode="contained"
+          />
```

Comment:
Minor thing but can you

- [ ] add the text "FloatingActionBar" to the FloatingActionBar
- [ ] have one button
- [ ] and make the bottom margin 100px

Those would make it a bit clearer that this is part of the UI Library and not a real part of the app.

Reply:
can do